NeoMathEngine/src/MathEngine.cpp

```diff
@@ -179,12 +176,7 @@ static CCudaDevice* captureCudaDevice( int deviceNumber, size_t deviceMemoryLimi
 
 		CCudaDevUsage dev;
 		dev.DevNum = i;
-		dev.Usage = 0;
-		for( int j = 0; j < CUDA_DEV_SLOT_COUNT; ++j ) {
-			if( !IsDeviceSlotFree( devProp.pciBusID, j ) ) {
-				++dev.Usage;
-			}
-		}
+		dev.Usage = GetDeviceUsage( devProp.pciBusID ) ;
```

Comment:
MB will be better than the count of blocks!

Reply:
fixed